common/src/main/java/org/apache/comet/parquet/NativeBatchReader.java

```diff
@@ -260,7 +260,8 @@ public void init() throws URISyntaxException, IOException {
     } ////// End get requested schema
 
     String timeZoneId = conf.get("spark.sql.session.timeZone");
```

Comment:
wondering should we always use UTC then and remove reading the tz from spark parameter?